recipes/molbox/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "molbox" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mosdef-hub/molbox/archive/{{ version }}.tar.gz
+  sha256: de3ff22c4c06fba472e7530468e1bae6efa3f51f86fb90a3b233ff6183f64d3b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+
+test:
+  imports:
+    - molbox
+
+about:
+  home: https://github.com/mosdef-hub/molbox
+  license: MIT
+  license_family: MIT
```

The full license mentions some code in a file borrowed from another project, so I *think* this should work:
```suggestion
  license: MIT AND BSD-3-Clause
```
Removed `license_family` since it's not mandatory and I have no idea what the right value is for this case.

---

Thanks. I have incorporated the suggestion.